webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -368,6 +368,16 @@ object RudderConfig extends Loggable {
     }
   }
 
+  val RUDDER_BCRYPT_COST = {
+    try {
+      config.getInt("rudder.bcrypt.cost")
+    } catch {
+      case ex: ConfigException =>
+        ApplicationLogger.info("Property 'rudder.bcrypt.cost' is missing or empty in rudder.configFile. Default cost to 12.")
```

Comment:
use debug here, because that information doesn't have any meaning safe if you use rudder-user plugin.